engines/core_by/app/models/core_by/user.rb

```diff
@@ -1,12 +1,14 @@
 # frozen_string_literal: true
 
 module CoreBy
-  class User < BaseRecord
+  class User < Base::ApplicationRecord
     extend AttachmentsVariants
 
     include SoftDeletable
     include WithExternalId
 
+    self.entity = Entities::User
```

Comment:
I think, it's worth using class names instead of classes for code reload compatibility, e.g. `self.entity_class ="Entities::User"`